packages/mui-material/src/Stepper/Stepper.test.tsx

```diff
@@ -258,4 +258,21 @@ describe('<Stepper />', () => {
     expect(stepContent[0]).not.to.have.class(stepContentClasses.last);
     expect(stepContent[1]).to.have.class(stepContentClasses.last);
   });
+
+  it('is applies non-linear styling', () => {
+    const { container, setProps } = render(
+      <Stepper nonLinear activeStep={0}>
+        <Step />
+        <Step />
+        <Step />
+      </Stepper>,
+    );
+
+    const stepper = container.querySelector(`.${classes.root}`);
+    expect(stepper).to.have.class(classes.nonLinear)
+
+    setProps({ alternativeLabel: true });
+
+    expect(stepper).to.have.class(classes.nonLinear)
```

Comment:
i believe these are not required, just testing presence of nonLinear class is required, which is what line `272` is doing.